TERN.md

```diff
@@ -797,6 +860,39 @@ a list.
 Many of these issues are discussed in [TURTLEDOVE Issue #38 - Contextual Bid](https://github.com/WICG/turtledove/issues/38)
 and [TURTLEDOVE Issue #37 - Clarification on Entities](https://github.com/WICG/turtledove/issues/37).
 
+### c. Contextual interest groups
```

Comment:
I don't understand this.  Does it accomplish something different than the original TURLTEDOVE proposal's way of adding people to interest groups from inside a 3p iframe?
> This could be on WeReallyLikeShoes.com, or could be a cross-domain iframe — maybe RunningShoeReviews.com writes articles about shoes sold by WeReallyLikeShoes.com, and the review site has an agreement which lets the retailer add people to an interest group..

Reply:
I think of it is a different implementation that is easier to accomplish, as it doesn't require any cross-domain iframes, or even, really, any coordination between advertisers and publishers. As per the example in this PR, it may not be very feasible to have iframes on all pages about video games to target the very large "video games" audience.

Reply:
First, I think this kind of thing _should_ require action by the publisher — see discussion in https://github.com/WICG/turtledove/issues/22.

Of course TERN is your document, so you can describe and propose whatever you want.  But this idea is definitely not without controversy.  I think you should make it clear what this provides that's different from what's in base TURTLEDOVE.

Second, If there is no cross-domain iframe, I don't see what keeps your competitors from messing with your interest groups.  (Either they can use your IG for their own advertising, or else they can maliciously add completely uninterested people to your IG without your consent.)

Reply:
What we find is that policies vary by SSP more than publisher, and the publisher can decide which SSPs to integrate with based on those policies. The addition of the `groups` object is not a _required_ field, and an integration between a DSP and SSP can specify whether that the object should be accepted.

My understanding was that interest groups would be namespaced by DSP, and these groups would be no different. Without that, it seems like anyone could muck with any interest group at any time. I would expect that SSPs would know that a response on the contextual side from a DSP would, indeed, be coming from that DSP with a preexisting trust relationship.

I'll find some time later today to write a commit that describes the above two items more fully, and also discusses the benefits we see over the base TURTLEDOVE proposal.